pkg/action/env.container_test.go

```diff
@@ -52,6 +52,7 @@ func prepareContainerTestSuite(t *testing.T) (*assert.Assertions, *gomock.Contro
 	r := &containerEnv{driver: d, dtype: "mock"}
 	r.AddImageBuildResolver(cfgImgRes)
 	r.SetContainerNameProvider(ContainerNameProvider{Prefix: containerNamePrefix})
+	r.SetActionsSum(newActionsSummary(launchrCfg()))
```

Comment:
Please, define the object near cfgImgRes. The same object is reused for all actions.